scripts/watchdog.ps1

```
@@ -0,0 +1,316 @@
+                return $proc
+            }
+        }
+        return $null
+    } catch {
+        return $null
+    }
+}
+
+function Stop-ProcessSafely {
+    param(
+        [int]$Pid
```

PSScriptAnalyzer (1.24.0)

[error] The Variable 'Pid' cannot be assigned since it is a readonly automatic variable that is built into PowerShell, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)